client/packages/common/src/intl/locales/en/common.json

```diff
@@ -9,33 +9,32 @@
   "button.add-item": "Add Item",
   "button.allocate": "Allocate",
   "button.cancel": "Cancel",
-  "button.save-and-confirm-status": "Confirm {{status}}",
   "button.close": "Close",
+  "button.create": "Create",
```

Comment:
Sorry... this was just putting everything in alphabetical order... I know @mark-prins likes the file this way 😉 ...